string taskID
---
uint8 status
string msg
